events/syscall/change_namespace_privillege_using_unshare.go

```diff
@@ -0,0 +1,43 @@
+//go:build linux
+// +build linux
+
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+    "syscall"
+
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(MaliciousProcessWithUnshare)
+
+func MaliciousProcessWithUnshare(h events.Helper) error {
+    if h.InContainer() {
+        cmd := exec.Command("/bin/sh", "-c", "unshare --fork --pid --mount-proc /bin/bash")
+        cmd.SysProcAttr = &syscall.SysProcAttr{
+			Cloneflags: syscall.CLONE_NEWNS | syscall.CLONE_NEWUSER,
+        }
+        
```

Comment:
I checked by running container on my host machine with alpine image and only running `unshare` command inside it will trigger rule so here we can change it to only command execution.
`cmd = exec.Command("unshare")`
provide your thoughts @leogr @FedeDP , Then i will change it.
![image](https://github.com/falcosecurity/event-generator/assets/117383853/f73abeb6-7913-474f-9d76-1877b0cce588)

Reply:
Agree, since the condition for the rule is:
```
 evt.type=unshare and evt.dir=< 
    and container 
    and not thread.cap_permitted contains CAP_SYS_ADMIN
```